unstructured_api_tools/pipelines/templates/pipeline_api.txt

```diff
@@ -29,11 +40,33 @@ async def pipeline_1(request: Request, file: UploadFile = File(),
 {{param}}: List[str] = Form(default=[]),
 {% endfor %}
 ):
-
+    
+    media_type = request.headers.get("accept")
+    {% if response_type %}
+    default_response_type = "{{response_type}}"
+    media_type = default_response_type if media_type == "*/*" else media_type
+    {% endif %}
     text = file.file.read().decode("utf-8")
+
+    {% if response_type %}
+    response = pipeline_api(text, media_type, {% for param in multi_string_param_names %}{{param}}, {% endfor %})
```

Comment:
This assumes that `response_type` is the second arg in the `pipeline_api` signature, but if I understand the JIRA ticket correctly, I think all we can assume is that `response_type` is a kwarg in the signature, so I think it would be safest to put it last in the signature with the form `response_type=media_type`.

Reply:
yeah that makes sense